documentation/src/docs/javadoc.md

```diff
@@ -60,7 +62,7 @@ maintain disparate storage systems - both KVTs and Streams share the same underl
 Pravega Segments.
 
 ## State Synchronizer Client
-_[State Synchronizer Client](https://github.com/pravega/pravega/blob/master/client/src/main/java/io/pravega/client/state/StateSynchronizer.java)_:
+_[State Synchronizer Client](https://www.pravega.io/docs/latest/javadoc/clients/io/pravega/client/state/StateSynchronizer.html)_:
 In many situations, a distributed application may need to share information across multiple processes that may
 update it (e.g., configuration file). In this scenario, keep the data consistent under concurrent updates is key.
 The State Synchronizer Client just solves this problem: it provides a means to have state that is synchronized between 
```

Comment:
The last link in this page should be pointing to https://www.pravega.io/docs/latest/javadoc/clients/index.html, but it is not working in your branch.

Reply:
Made the changes.